### ISSUE TITLE
Pass session token id to /password/change/finish, not raw token bytes

### ISSUE THREAD

**Issue by rfk** [mozilla/fxa-content-server#3875]
A follow-up to https://github.com/mozilla/fxa-auth-server/pull/1302

The way that we currently pass a sessionToken to the `/password/change/finish` endpoint, is by sending the raw data bytes in the request body. This is the only place in the code that the client sends session data in raw form over the wire - in all other places, we derive a "token id" and "token key" and use those to make a signed request in a way that's resistant to a MITM attack.

Let's change the client code to send the id instead of the raw token data, to keep the security model of these things consistent. Auth-server support for this will land in train-65, so we will need to wait until at least train-66 to make the change on the content-server side.

Once landed, we'll need a follow-up issue on the auth-server to stop *accepting* raw token bytes, which it currently still does for backwards-compatibility reasons.

**Comment by vbudhram** [comment]
Per chat with @rfk : the magic sauce to get session id from session data located here https://github.com/mozilla/fxa-js-client/blob/master/client/lib/hawkCredentials.js#L19, ref diagram: https://github.com/mozilla/fxa-auth-server/wiki/onepw-protocol#signing-certificates

**Comment by vladikoff** [comment]
blocked by : train deploys, waiting for train-66 cut
